src/lib.rs

```diff
@@ -1,68 +1,77 @@
 //! Audio playback library.
 //!
-//! The main concept of this library is [the `Source` trait](source/trait.Source.html), which
+//! The main concept of this library is the [`Source`] trait, which
 //! represents a sound (streaming or not). In order to play a sound, there are three steps:
 //!
 //! - Create an object that represents the streaming sound. It can be a sine wave, a buffer, a
-//!   [decoder](decoder/index.html), etc. or even your own type that implements
-//!   [the `Source` trait](source/trait.Source.html).
-//! - Choose an output with the [`devices`](fn.devices.html) or
-//!   [`default_output_device`](fn.default_output_device.html) functions.
-//! - Call [`play_raw(output, source)`](fn.play_raw.html).
+//!   [`decoder`], etc. or even your own type that implements the [`Source`] trait.
+//! - Get an output stream handle to a physical device. For example, get a stream to the system's
+//!   default sound device with [`OutputStream::try_default()`]
+//! - Call [`.play_raw(source)`](OutputStreamHandle::play_raw) on the output stream handle.
 //!
-//! The `play_raw` function expects the source to produce `f32`s, which may not be the case. If you
-//! get a compilation error, try calling `.convert_samples()` on the source to fix it.
+//! The [`play_raw`](OutputStreamHandle::play_raw) function expects the source to produce [`f32`]s,
+//! which may not be the case. If you get a compilation error, try calling
+//! [`.convert_samples()`](Source::convert_samples) on the source to fix it.
 //!
 //! For example, here is how you would play an audio file:
 //!
-//! ```no_run
+//! ```
```

Comment:
Travis CI failed... maybe add back the no_run here?

Reply:
✅ Sure thing